includes/Assets.php

```diff
@@ -578,16 +588,43 @@ public function dokan_dashboard_scripts() {
             wp_enqueue_script( 'jquery-ui-datepicker' );
             wp_enqueue_script( 'underscore' );
             wp_enqueue_script( 'post' );
-            wp_enqueue_script( 'dokan-tooltip' );
+
+            if (
+                isset( $wp->query_vars['products'] ) ||
+                isset( $wp->query_vars['orders'] ) ||
+                isset( $wp->query_vars['coupons'] ) ||
+                isset( $wp->query_vars['reports'] ) ||
+                isset( $wp->query_vars['return-request'] ) ||
+                isset( $wp->query_vars['support'] ) ||
+                ( isset( $wp->query_vars['settings'] ) && in_array( $wp->query_vars['settings'], [ 'store', 'shipping', 'regular-shipping', 'seo' ], true ) )
+            ) {
+                wp_enqueue_script( 'dokan-tooltip' );
+            }
+
             wp_enqueue_script( 'dokan-form-validate' );
-            wp_enqueue_script( 'dokan-tabs' );
-            wp_enqueue_script( 'dokan-chart' );
-            wp_enqueue_script( 'dokan-flot' );
+
+            if ( isset( $wp->query_vars['products'] ) || isset( $wp->query_vars['tools'] ) ) {
+                wp_enqueue_script( 'dokan-tabs' );
+            }
+
+            if (
+                isset( $wp->query_vars['page'] ) ||
+                isset( $wp->query_vars['products'] ) ||
+                isset( $wp->query_vars['reports'] )
+            ) {
+                wp_enqueue_script( 'dokan-chart' );
+                wp_enqueue_script( 'dokan-flot' );
+            }
+
             wp_enqueue_script( 'dokan-select2-js' );
             wp_enqueue_media();
             wp_enqueue_script( 'dokan-accounting' );
             wp_enqueue_script( 'serializejson' );
-            wp_enqueue_script( 'dokan-popup' );
+
+            if ( isset( $wp->query_vars['products'] ) || isset( $wp->query_vars['coupons'] ) ) {
+                wp_enqueue_script( 'dokan-popup' );
```

Comment:
Please recheck, this popup is workable with entire plugin again both light and pro.
Also is coupons page necessary this dokan-popup ?

Reply:
some library (like sweetalert2) has dependency on 'dokan-popup', those library will load it as necessary